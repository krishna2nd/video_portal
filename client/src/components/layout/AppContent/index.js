import React, { Component } from 'react';
import './index.css';
class AppContent extends Component {
  render() {
    return (
      <article>
        <div className="container" >
          {
            this.props.children
          }
        </div>
      </article>
    );
  }
}

export default AppContent;
