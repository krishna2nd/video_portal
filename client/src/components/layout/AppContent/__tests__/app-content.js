import React from 'react';
import AppContent from '../';
import renderer from 'react-test-renderer';

test('components/AppContent', () => {
  const component = renderer.create(
    <AppContent >
      <p>Content..</p>
    </AppContent>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});