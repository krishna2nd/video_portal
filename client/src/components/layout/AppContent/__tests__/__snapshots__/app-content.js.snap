// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AppContent 1`] = `
<article>
  <div
    className="container"
  >
    <p>
      Content..
    </p>
  </div>
</article>
`;
